### PR TITLE
Expose contact meta data on Contact detail view

### PR DESCRIPTION
**Pull request by hodgestar**
The only way to see the contact created at information at the moment is via the API, ideally it would be shown in the contact detail view

### CODE REVIEW

File: go/contacts/forms.py [hunk 1]
```diff
@@ -19,8 +19,8 @@ class ContactForm(forms.Form):
     def __init__(self, *args, **kw):
         groups = kw.pop('groups')
         super(ContactForm, self).__init__(*args, **kw)
-        self.fields['groups'] = forms.MultipleChoiceField(required=False,
-            choices=[(g.key, g.name) for g in groups])
+        self.fields['groups'] = forms.MultipleChoiceField(
+            required=False, choices=[(g.key, g.name) for g in groups])
```
Comment (hodgestar) [inline go/contacts/forms.py:23]: PEP8 fix.

File: go/contacts/forms.py [hunk 2]
```diff
@@ -43,9 +43,10 @@ class SmartGroupForm(forms.Form):
 
 
 class UploadContactsForm(forms.Form):
-    file = forms.FileField(label="File with Contact data",
+    file = forms.FileField(
+        label="File with Contact data",
         help_text="This can either be a double-quoted UTF-8 encoded CSV file "
-                    "or an Excel spreadsheet")
+                  "or an Excel spreadsheet")
```
Comment (hodgestar) [inline go/contacts/forms.py:49]: PEP8 fix.

File: go/contacts/views.py [hunk 3]
```diff
@@ -427,7 +427,7 @@ def _people(request):
     # production hack if we really need to count all contacts or something.
     fetch_limit = int(request.GET.get('_fetch_limit', 10000))
     if query:
-        if not ':' in query:
+        if ':' not in query:
```
Comment (hodgestar) [inline go/contacts/views.py:430]: Flake8 recommended style fix.

**Comment by hodgestar** [comment]
@imsickofmaps Ready for review.

**Comment by imsickofmaps** [comment]
:+1: on the meta data stuff. The subscriptions bit looks correct to me but I don't know the inner workings enough to know if you've mocked reality correctly :grinning:

**Comment by hodgestar** [comment]
Subscriptions are functionally identical to extras, they just have a
different name and we store different things in them.

**Pull request merged**